docs/source/specification/quarterly_roadmap.md

```diff
@@ -21,52 +21,65 @@
 
 A quarterly roadmap will be published to give the DataFusion community visibility into the priorities of the projects contributors. This roadmap is not binding.
 
-## 2022 Q1
+## 2022 Q2
 
 ### DataFusion Core
 
-- Publish official Arrow2 branch
-- Implementation of memory manager (i.e. to enable spilling to disk as needed)
+- IO Improvements
+  - Reading, registering, and writing more file formats from both DataFrame API and SQL
+  - Additional options for IO including partitioning and metadata support
+- Memory Management
```

Comment:
```suggestion
- Work Scheduling
  - Improve predictability, observability and performance of both IO and CPU-bound work
  - Develop a more explicit story for managing parallelism during plan execution
- Memory Management
```

I've yet to create a ticket for this, as I'm still exploring the problem domain, but the precursor discussions can be found https://github.com/apache/arrow-rs/issues/1473 and https://github.com/apache/arrow-datafusion/issues/2079.